using-list-methods

```diff
@@ -0,0 +1,9 @@
+# In python we gt many options to combine elements of list into single string.
+
+arr = ['H','e','l','l','o',' ','W','o','r','l','d']   # initiating the list with elements
+
+print("".join(arr))  # using join method with double quote
+print(''.join(arr))  # using join method with single quote
+print(*arr,sep='')   # using * operator with sep attribute
+print(''.join(arr))  # using join method
+print(''.join(map(str, arr)))  # using join and map methods to convert into single string
```

Comment:
This method of using map to join items in a list has also been [implemented](https://github.com/Goku-kun/1000-ways-to-print-hello-world-in-python/blob/master/using-map().py).

Reply:
After going through the code mentioned [here](https://github.com/Goku-kun/1000-ways-to-print-hello-world-in-python/blob/master/using-map().py), the code relies on only 2 elements of array. If different ways is there, I'll suggest to look into it as 2 words (i.e. Hello & World) & 10 letter (i.e. H,E,L,L,O,W,O,R,L,D). Try to think like this & revert back.

Reply:
Haha. If we think along those lines, by using combinations we could show that there are more than at least 10 different ways to print out "Hello, World!" with simply different concatenation variations. I don't believe those should be considered very different. Let me see if I can explain it with an illustration. 2+2 == 4 and 1+3 == 4. Are these implementations different? Yes. But, they're employing the same functional logic in my opinion.

Reply:
In that case, you can leave it. Thanks !